backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,11 +170,15 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
     private String getDiskIsBeingMigratedMessage() {
         if (cachedDiskIsBeingMigratedMessage == null) {
+            DiskImage diskImage = diskImageDao.get(getParameters().getImageId());
+            String diskName = "";
+            if (diskImage != null && diskImage.getDiskAlias() != null) {
+                diskName = diskImage.getDiskAlias();
+            }
```

Comment:
@ArtiomDivak previously we set `getParameters().getNewAlias()` which apparently can be null in some flow and now you'll take it from `diskImage.getDiskAlias()` or set an empty string. so now we never use `getParameters().getNewAlias()` which seems incorrect. how about still using `getParameters().getNewAlias()` if not null or `diskImage.getDiskAlias()`? (there's no need to set it to empty string, see [this](https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.5.0.8/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/BaseDisk.java#L59) and [this](https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.5.0.8/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/BaseDisk.java#L181)

Reply:
I think in MoveDiskCommand `getParameters().getNewAlias()` is never set as there is no option to change the alias

https://github.com/oVirt/ovirt-engine/blob/997916583d9c7a2e7bdb7b51c6ae514426752c47/backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendDiskResource.java#L113